sos/plugins/networking.py

```diff
@@ -207,7 +215,23 @@ def setup(self):
                         or line.isspace() \
                         or line[:1].isspace():
                     continue
-                out_ns.append(line.partition(' ')[0])
+                # if namespaces defined, remove from out_ns not matching
+                # namespaces with regex
+                if self.get_option("namespaces"):
+                    regex = '(?:%s)' % '|'.join(
+                        self.get_option("namespaces").split())
+                    if bool(match(regex, line)):
+                        out_ns.append(line.partition(' ')[0])
+
+                # if inspect_namespaces is defined, remove from out_ns
+                # namespaces with higher index than defined
+                elif self.get_option("inspect_namespaces") != 0:
+                    out_ns.append(line.partition(' ')[0])
+                    if len(out_ns) == self.get_option("inspect_namespaces"):
+                        break
```

Comment:
This wont work when using both `namespaces` and `inspect_namespaces`: isnt it easier to add at the end of whole cycle `if len(out_ns) == get_option(..) then break`?

Reply:
Ah that is intentional that the options can be used exclusively only. Taking back this, just not sure if the info about exclusive suage of the options shall not go to code or help.

Reply:
This works how was proposed in #1916 , inspect_namespaces will work only when namespaces are not defined. I have added comment into the code and added better parameter description.